components/avatar/avatar.stories.js

```
@@ -82,12 +73,25 @@ export const argTypesData = {
   overlayClass: {
     description: 'Pass through classes. Used to customize the avatar overlay',
   },
+  iconName: {
+    options: iconsList,
+    control: {
+      type: 'select',
+      labels: {
+        undefined: '(empty)',
+      },
+    },
+  },
+  iconSize: {
+    defaultValue: null,
+  },
```

there are props in the component were are not documented here, also could we sort alphabetically props here and in the component please? so many props its kind of messy

---

We shouldn't include all the properties there, only the ones that need set default values, hiding them from the panel and so forth, using JSDoc on the component itself it's enough for storybook and for our documentation extractor to work properly. I'll alphabetically sort them.